config/config.redaction.yaml

```diff
@@ -0,0 +1,28 @@
+keys:
+  - administration.readerGroupId
```

Comment:
Crazy thought - why not every UUID?

Reply:
e.g. use the regex support in the sanitizer to just nuke all UUID-looking things. The approach here will be forever-whack-a-mole.

Reply:
Also, wouldn't customer subscription IDs, etc, be considered PII?

Reply:
Is it only UIDs we want to redact? Let me try figure that out. If so, we can do a uid regex

Reply:
What about log correlation IDs? I think simply filtering all uids won't work in that case